test/testSetup.ts

```diff
@@ -113,23 +115,27 @@ export class TestSetup {
 
     // If you are not in a VSCode project, the Salesforce extensions are not running
     // Force the CLI integration extension to load before creating the project
-    const workbench = await (await browser.getWorkbench()).wait();
-    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Show Running Extensions', 5);
-    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Project', 5);
+    const workbench = await utilities.getWorkbench();
+    await utilities.showRunningExtensions();
+    const prompt = await workbench.executeQuickPick('SFDX: Create Project');
+    await utilities.waitForQuickPick(prompt, 'Standard', {
+      msg: 'Expected extension salesforcedx-core to be available within 5 seconds',
+      timeout: 5_000
+    });
     await browser.keys(['Escape']);
     await utilities.pause(1);
     await browser.keys(['Escape']);
 
-    // Do not continue until we verify CLI Integration extension is present and running
-    let coreExtensionWasFound = false;
-    do {
-      await utilities.pause(10);
+    const coreIsActive = await utilities.verifyExtensionsAreRunning(
+      utilities
+        .getExtensionsToVerifyActive()
+        .filter((ext) => ext.extensionId === 'salesforcedx-vscode-core')
+    );
+
+    if (!coreIsActive) {
+      fail('Expected core extension to be active after 20 seconds');
+    }
 
-      coreExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
-        workbench,
-        'salesforcedx-vscode-core'
-      );
-    } while (coreExtensionWasFound === false);
     utilities.log(`${this.testSuiteSuffixName} - Ready to create the standard project`);
 
     await this.createProject(workbench, this.tempProjectName, scratchOrgEdition);
```

Comment:
`createProject()` only expects 2 arguments.

```suggestion
    await this.createProject(this.tempProjectName, scratchOrgEdition);
```